libbeat/docs/event-conventions.asciidoc

```diff
@@ -1,37 +1,47 @@
 [[event-conventions]]
-=== Event conventions
-
-==== Naming conventions
-
-For the field names, the Beats follow the naming conventions below:
-
-- All fields must be lower case
-- Snake case for combining words
-- Group related fields in sub documents, which means using the . notation. Groups are mostly described by common prefixes.
-- Prevent namespace duplication. If connections appears in the namespace, it's not needed in the sub document
-- Do not use complex abbreviations. A list of standardised abbreviations can be found below.
-- Organise the documents from the general to the details, which allows namespacing. The type should always be last, like .pct.
-- If two fields are the same but with different units, remove the less granular one
+=== Naming Conventions
+
+When creating events, use the following conventions for field names and abbreviations.
+
+[[field-names]]
+==== Field Names
+
+Use the following naming conventions for field names:
+
+- All fields must be lower case.
+- Use snake case (underscores) for combining words.
+- Group related fields into subdocuments, which means using the dot (.) notation. Groups are mostly described by common prefixes. 
+//QUESTION: I am not sure what you mean by common prefixes here. Perhaps examples would help?
+- Prevent namespace duplication. If connections appears in the namespace, it's not needed in the sub document 
+//QUESTION: I don't understand what you mean here. Can you clarify or give an example?
```

Comment:
Same example as above. Instead of going with
```
cpu.cpu_load
cpu.cpu_system
```

The cpu part is removed as it already shows up in the namespace